packages/chat/react/feedback.ts

```diff
@@ -1,19 +0,0 @@
-/**
```

Comment:
Can you put both the react/feedback.ts and react/test-input.ts files back in?

Reply:
I pulled in with git fetch upstrean but don't see them, the version I pushed before was the latest so unsure where these files are, are they auto generated?

Reply:
I belive anna put them in manually. You should be able to just copy and paste the contents from github where it shows they were removed and push a commit with the files again